### PR TITLE
iOS Drawer header fix and renaming of dateUtils and domUtils

### PR DESCRIPTION
**Pull request by morethanfire**

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -39,8 +39,8 @@ export { default as TimePicker } from './modules/timePicker.js';
 
 // utils
 export { default as DatePickerUtils } from './utils/datePickerUtils.js';
-export { default as DateUtils } from './utils/dateUtils.js';
-export { default as DOMUtils } from './utils/domUtils.js';
+export { default as dateUtils } from './utils/dateUtils.js';
```
Comment (groberts314) [inline src/index.js]: Gonna be a find/replace for the imports in HC I imagine?

File: src/modules/drawer.js [hunk 2]
```diff
@@ -369,20 +299,85 @@ class Drawer extends React.Component {
...
+                <div
+                    className={drawerClasses}
+                    ref={(ref) => { this.drawerRef = ref; }}
+                >
+                    <div
+                        className="drawer-container"
+                        ref={(ref) => { this.drawerContainerRef = ref; }}
+                        style={{
+                            height: _.isNumber(positionYOffset) ? `calc(100% - ${positionYOffset}px)` : null,
+                            // transform: this.setStartOfTransform(),
```
Comment (groberts314) [inline src/modules/drawer.js]: Is there a good reason to keep this commented out?

File: src/index.js (hunk 1 above)
Comment (morethanfire) [reply src/index.js]: Yeah. I thought about not doing this change just yet, but I don't believe it'll be that time consuming go forth.

File: src/modules/drawer.js (hunk 2 above)
Comment (morethanfire) [reply src/modules/drawer.js]: removed.

**Comment by groberts314** [comment]
:shipit:

**Pull request merged**